propka/protonate.py

```diff
@@ -407,31 +408,26 @@ def set_bond_distance(self, a, element):
         if element in list(self.bond_lengths.keys()):
             d = self.bond_lengths[element]
         else:
-            print('WARNING: Bond length for %s not found, using the standard value of %f'%(element, d))
+            info_warning('Bond length for %s not found, using the standard value of %f' % (element, d))
 
         a = a.rescale(d)
 
         return a
 
     def display(self,*text):
```

Comment:
If you want, you could replace all calls to display() with debug(), and remove display() completely.